template.yaml

```diff
@@ -0,0 +1,1157 @@
+    Description: "The ARN of the permissions boundary to apply when creating IAM roles"
+    Type: String
+    Default: "none"
+  MaxContainerCount:
+    Description: Maximum number of FE node/express container tasks to allow autoscaling to reach
+    Type: Number
+    Default: 12
+  MinContainerCount:
+    Description: Minimum number of FE node/express container tasks to allow autoscaling to reach
+    Type: Number
+    Default: 3
+  EnableScalingInDev:
+    Type: Number
+    Default: 0
+    Description: Set to 1 to enable deployment of scaling infra in dev
+  IPVStubStackName:
+    Type: String
+    Default: "bav-ipv-stub"
+    Description: The name of the deployed IPV stub stack.
+
+Conditions:
+  IsNotDevelopment: !Or
+    - !Equals [ !Ref Environment, build ]
+    - !Equals [ !Ref Environment, staging ]
+    - !Equals [ !Ref Environment, integration ]
+    - !Equals [ !Ref Environment, production ]
+  EnableScaling: !Or
+    - !Equals [ !Ref EnableScalingInDev, 1]
+    - !Equals [ !Ref Environment, build ]
+    - !Equals [ !Ref Environment, staging ]
+    - !Equals [ !Ref Environment, integration ]
+    - !Equals [ !Ref Environment, production ]
+  UsePermissionsBoundary:
+    Fn::Not:
+      - Fn::Equals:
+          - !Ref PermissionsBoundary
+          - "none"
+  IsProdLikeEnvironment: !Or
+    - !Equals [!Ref Environment, staging]
+    - !Equals [!Ref Environment, integration]
+    - !Equals [!Ref Environment, production]
+  IsNotProdLikeEnvironment: !Not
+    - !Condition IsProdLikeEnvironment
+
+Mappings:
+  PlatformConfiguration:
+    dev:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    build:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    staging:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    integration:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    production:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+
+# see https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+  ElasticLoadBalancerAccountIds:
+    eu-west-2:
+      AccountId: 652711504416
+
+  EnvironmentVariables:
+    ### These environment variables are referenced further down the file.
+    ### Please ensure that any variables defined for an environment are defined for _all_ environments.
+    dev:
+      EXTERNALWEBSITEHOST: "https://cic-cri-front.review-c.dev.account.gov.uk"
+      APIBASEURL: "https://api-cic-cri-api.review-c.dev.account.gov.uk" 
+      DNSSUFFIX: "review-c.dev.account.gov.uk"
+      SESSIONTABLENAME: "cic-front-sessions-dev"
+      GTMIDGA4: "GTM-KD86CMZ"
+      ANALYTICSDOMAIN: "dev.account.gov.uk"
+      TESTHARNESSURL: "https://cic-test-harness-testharness.review-c.dev.account.gov.uk"
```

Comment:
These values should all exist now for BAV (at least in dev) let me know if you need a hand finding them

Reply:
Thanks! All added